app/src/main/java/org/p2p/wallet/home/ui/main/HomePresenter.kt

```diff
@@ -80,8 +85,12 @@ class HomePresenter(
 
     init {
         // TODO maybe we can find better place to start this service
+        val userSeedPhrase = seedPhraseProvider.getUserSeedPhrase().seedPhrase
         launch {
             awaitAll(
+                async {
+                    ethereumRepository.init(seedPhrase = userSeedPhrase)
+                },
```

Comment:
I can't see the feature toggle for it. 
We'll need to release swaps and bridges will not be ready at that time